tests/pytorch/test_unified_tensor.py

```diff
@@ -1,85 +0,0 @@
-import dgl.multiprocessing as mp
```

Comment:
I understand `contrib.UnifiedTensor` has been merged to the underlying of DataLoader. Are these test cases covered by other unit tests? If not, we should migrate them first.

Reply:
I believe they have been covered by the `uva_cpu_indices` and `uva_cuda_indices` options of the dataloader tests.
https://github.com/dmlc/dgl/blob/00f2099934ca78a3c156c4222edcc3245cee4e1c/tests/pytorch/test_dataloader.py#L172